app/views/tools/_form.html.erb

```diff
@@ -5,7 +5,7 @@
                 <%= form.label :auth_url, "Auth URL", class: "usa-label usa_textarea" %>
                 <%= form.text_area :auth_url, class: "block col-12 optional usa-textarea", required: true
                 %>
-                <div class="usa-form-hint">Paste the entire URL of the auth request. If the signature is included in the SAML itself, you can also just paste the contents of the SAMLRequest parameter.</div>
+                <div class="usa-form-hint">Paste the entire URL of the auth request, which you can capture using your <a href="https://en.wikipedia.org/wiki/Web_development_tools" target="_blank">browser's web developer tools</a>. If you are using <code>POST</code> instead of <code>GET</code>, or the signature is included in the SAML itself, you can also just paste the contents of the SAMLRequest parameter.</div>
```

Comment:
```suggestion
                <div class="usa-form-hint">Paste the entire URL of the auth request, which you can capture using your <a href="https://en.wikipedia.org/wiki/Web_development_tools" target="_blank">browser's web developer tools</a>. If you are using <code>POST</code> instead of <code>GET</code>, or if the signature is included in the SAML itself, you can also just paste the contents of the SAMLRequest parameter.</div>
```

The "if" got lost in this message when you added the additional help text.